inc/class-saml.php

```diff
@@ -121,7 +121,7 @@ static public function hooks( SAML $obj ) {
 		add_filter( 'authenticate', [ $obj, 'authenticate' ], 10, 3 );
 		add_action( 'login_enqueue_scripts', [ $obj, 'loginEnqueueScripts' ] );
 		add_action( 'login_form', [ $obj, 'loginForm' ] );
-		add_filter( 'logout_redirect', [ $obj, 'logoutRedirect' ] );
+		add_filter( 'init', [ $obj, 'logoutRedirect' ] );
```

Comment:
I used `init` hook, since others like `logout_redirect` and `wp_logout` are triggered after the session has been terminated.
With this hook we can use `$_SESSION` global var, which was the one used before the latest release that included the cookie usage, causing problems with some clients. I could not replicate the issues reported, however I am guessing that using sessions intead of cookies as before, it'd be fix it.